pkg/cmd/util/cmd.go

```diff
@@ -163,3 +163,11 @@ func VersionedPrintObject(fn func(*cobra.Command, meta.RESTMapper, runtime.Objec
 		return fn(c, mapper, obj, out)
 	}
 }
+
+func WarnAboutCommaSeparation(errout io.Writer, values []string, flag string) {
+	for _, value := range values {
+		if strings.ContainsRune(value, ',') {
+			fmt.Fprintf(errout, "warning: %s no longer accepts comma-separated list of values. %q will be treated as a single key-value pair.\n", flag, value)
```

Comment:
we should only warn if the value looks like "key1=value1,key2=value2".  In other words, we have to see both a command and an equal after key1= for it to be a problem.  There's nothing wrong with "key1=a,b,c"